install.sh

```diff
@@ -11,7 +11,7 @@ sh ./configure --prefix /usr
 sudo apt-get install libx11-dev
 make
 make DESTDIR="/tmp/xrectsel" install
-sudo cp xrectsel /usr/bin/xrectsel
+sudo cp usr/bin/xrectsel /usr/bin/xrectsel
```

Comment:
Thanks for the contribution :heart: 

```suggestion
sudo cp /usr/bin/xrectsel /usr/bin/xrectsel
```

You forgot the first `/` (slash)